### PR TITLE
Simplify common autoyast/create_hdd/toolchain tests for openSUSE/SLE

### PR DESCRIPTION
**Pull request by okurz**
Verification log output from local run of openSUSE toolchain tests:

```
scheduling boot_to_desktop tests/boot/boot_to_desktop.pm
scheduling force_cron_run tests/console/force_cron_run.pm
scheduling install tests/toolchain/install.pm
scheduling gcc_fortran_compilation tests/toolchain/gcc_fortran_compilation.pm
scheduling gcc_compilation tests/toolchain/gcc_compilation.pm
```

### CODE REVIEW

File: products/opensuse/main.pm [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # SUSE's openQA tests
 #
 # Copyright © 2009-2013 Bernhard M. Wiedemann
-# Copyright © 2012-2016 SUSE LLC
+# Copyright © 2012-2017 SUSE LLC
```
Comment (asmorodskyi) [inline products/opensuse/main.pm]: it will be valid for 12 days only :)

**Comment by jknphy** [comment]
May I ask you why you call it common? There is nothing else in the file common between openSUSE and SLE15? I think that with a function for the toolchain would be enough.

**Comment by asmorodskyi** [comment]
@jknphy sorry I press "Merge" before your comment appear . I think there are a lot in common , but there is long story behind with spiting and  merging test codebase between SLE and openSUSE. 
@okurz just trying to merge back this common thing again and ofcourse you can't do all this in one merge request

**Comment by jknphy** [comment]
no problem, thanks for realizing, I was wondering if the word "common" would not be confusing, but the proposal code is better grouped, so LGTM.

**Pull request merged**